### DISCUSSION TITLE
Alpha Mask not working on AR in android / Samsung

### DISCUSSION THREAD

**Discussion by felipefierro** [google/model-viewer#4313] [Q&A] [answered]
Hi! the alpha is not working in AR on a Samsung / Android device, but in an Apple I'OS device it's working perfectly:

![palm2](https://github.com/google/model-viewer/assets/134464697/96ad5a9b-dbe1-4c13-81b9-bc9a44386fea)
![palm](https://github.com/google/model-viewer/assets/134464697/ef4fda42-8238-43bc-ae72-5a0e69973b79)

**Comment by felipefierro** [comment]
Also the 
![imagen](https://github.com/google/model-viewer/assets/134464697/ac1676f4-41ec-448f-8620-1cb67b1b775c)

isn't working on Iphone:

`    <model-viewer src="escena.glb" ar ar-modes="webxr scene-viewer quick-look" disable-tap  auto-rotate ar-scale="fixed" disable-zoom  camera-controls poster="poster.webp" disable_occlusion="true" shadow-intensity="1"  max-camera-orbit="auto ">`

**Reply by elalish** [reply, marked as answer]
Oh, actually I shouldn't have converted this one to a discussion. This is an issue I just fixed upstream in three.js, so it should be better soon when we merge in the new version. I was thrown off by `disable_occlusion`: I'm not aware of that being part of our API. https://modelviewer.dev/docs/index.html

**Reply by felipefierro** [reply]
Don't worry! thank you so much! any news on this release?

**Reply by elalish** [reply]
working on it - should be sometime this month.
